src/TwitchLib.Communication/Events/OnSendFailedEventArgs.cs

```diff
@@ -1,16 +1,14 @@
-﻿using System;
+﻿namespace TwitchLib.Communication.Events;
 
-namespace TwitchLib.Communication.Events
+public class OnSendFailedEventArgs : EventArgs
 {
-    public class OnSendFailedEventArgs : EventArgs
-    {
-        public string Data { get; }
-        public Exception Exception { get; }
+    public string Data { get; }
```

Comment:
we use `Message` in `OnMessageEventArgs`, and the type here is still string. Should this be `Message` instead of `Data`?

Reply:
I don't have a strong opinion on it, so I can change it if you want

Reply:
I would keep it consistent yeah. either change both to Data or both to Message